### PR TITLE
Added form validation

### PR DESCRIPTION
**Pull request by ManarOmari**
For this milestone of the portfolio website I

-  implement a client-side validation of the data in the contact form before submission.

### CODE REVIEW

File: form_validation.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+const validateEmail = (email) => {
+  if (email.match(/^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9-]+(?:\.[a-z0-9-]+)*$/)) {
+    return true;
+  }
+  return false;
+};
+
+submitBtn.addEventListener('click', () => {
+  const emailVal = document.getElementById('mail').value;
+  if (!validateEmail(emailVal)) {
```
Comment (dasileker) [inline form_validation.js]: - [ ] Hello and nice work on adding the form however the validation process required is not quite the one like you have implemented, I suggest visiting the requirement page again and reading it.

Capture web_6-9-2022_12576_github com

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @ManarOmari,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**
- nice Implementation 👌.
- 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: form_validation.js (hunk 1 above)
Comment (ManarOmari) [reply form_validation.js]: solved

**Review by phelian23 — APPROVED** [review]
Hi @ManarOmari,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f5061536e555a573145657970756c734f77642f67697068792e676966

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**